_dashboards/management/connect-prometheus.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Connecting Prometheus to OpenSearch
+parent: Data sources
+nav_order: 20
+---
+
+# Connecting Prometheus to OpenSearch
```

Comment:
We need to add visuals to this documentation. It is end user doc and dashboards related. See http://localhost:4000/docs/latest/dashboards/management/S3-data-source/ for an example.